frontend/packages/erd-core/src/features/gtm/utils/openRelatedTablesLogEvent.ts

```diff
@@ -0,0 +1,14 @@
+import { pushToDataLayer } from './pushToDataLayer'
+
+type OpenRelatedTablesLogEvent = {
+  tableId: string
+}
+
+export const OpenRelatedTablesLogEvent = ({
```

Comment:
[nits]
I thought it might be better to make the first letter of function names lowercase.

```suggestion
export const openRelatedTablesLogEvent = ({
```

Reply:
Thank you for suggestion! 👍